src/components/calcite-color-picker/calcite-color-picker.tsx

```diff
@@ -607,7 +609,9 @@ export class CalciteColorPicker {
   /** Sets focus on the component. */
   @Method()
   async setFocus(): Promise<void> {
-    focusElement(this.hexInputNode);
+    const elementToFocus = getFocusableElements(this.el.shadowRoot)[0] || this.el;
```

Comment:
No need to query, let's just focus on the scope ref node. 

Sidebar, looks like `focusElement(host)` will cause an infinite loop based on the logic. Do you think renaming the parameter would help? Otherwise, we may need to refactor to `focusElement(host, target)`. We can tackle in a separate issue.

Reply:
ah, yeah good catch. I'll remove.

> No need to query, let's just focus on the scope ref node.

Ok. We can revisit after we get feedback from level access.